openmc/model/model.py

```diff
@@ -1142,38 +1162,63 @@ def differentiate_depletable_mats(self, diff_volume_method: str):
             volume equally between the new materials, 'match cell' sets the
             volume of the material to volume of the cell they fill.
         """
+        self.differentiate_mats(diff_volume_method, depletable_only=True)
+
+    def differentiate_mats(self, diff_volume_method: str = None, depletable_only: bool = True):
+        """Assign distribmats for each depletable material
+
+        .. versionadded:: 0.14.0
```

Comment:
We'll need to update the version here at this point now that we're at `0.15.1-dev` now.